cms/jinja2/component_overrides/header/_macro.njk

```diff
@@ -137,6 +138,9 @@
                                     }}
                                 </div>
                             {% endif %}
+                            <div class="ons-grid__col ons-col-auto">
+                                <a href={{ otherLanguage.url }}>{{ otherLanguage.text }}</a>
+                            </div>
```

Comment:
thought: it would be nice to make this work with any number of "other languages", but perhaps that is a premature optimisation at this point 🤔

Reply:
Fair point, addressed in [f732e77](https://github.com/ONSdigital/dis-wagtail/pull/140/commits/f732e77f73c421a5f1473c57c237c7ddbd548cb7)